scripts/generate-zeus.sh

```diff
@@ -12,7 +12,7 @@ function generate() {
   local MV_PATH=$GEN_PATH/$1; shift
 
   # pass the rest of the arguments to zeus
-  zeus $HASURA_GRAPHQL_ENDPOINT $GEN_PATH --ts --rq $@
+  (set -x; zeus $HASURA_GRAPHQL_ENDPOINT/v1/graphql $GEN_PATH --ts --rq $@)
```

Comment:
Why did this change?

Reply:
because it was wrong, from when I wrote it

Reply:
that's weird, it was working for me as it was--i ran this several times to test it when i refactored it

Reply:
weird, maybe it was something with renaming variables actually

Reply:
hmm. ideally we wouldn't leave this `/v1/graphql` bit here when we have it in the env variable in other cases, for consistency's sake